assets/assets.go

```diff
@@ -11,3 +16,29 @@ var f embed.FS
 func ReadFile(name string) ([]byte, error) {
 	return f.ReadFile(name)
 }
+
+func RelatedObjects(rm meta.RESTMapper, files []string) ([]configv1.ObjectReference, error) {
+	relatedObjects := make([]configv1.ObjectReference, 0, len(files))
+	for _, f := range files {
+		data, err := ReadFile(f)
+		if err != nil {
+			return nil, err
```

Comment:
What do you think about wrapping the all the errors we return in this func? Future-us might thank us for making it easier to narrow down the exact location of the error?